web/src/lib/components/user-settings-page/user-settings-list.svelte

```diff
@@ -67,3 +68,7 @@
 <SettingAccordion title="Sidebar" subtitle="Manage sidebar settings">
   <SidebarSettings />
 </SettingAccordion>
+
+<SettingAccordion title="Appearance" subtitle="Manage the appearance of Immich">
```

Comment:
Can you put this at the top to keep things alphabetical?

Reply:
yep sure, done